### ISSUE TITLE
Workload service account

### ISSUE THREAD

**Issue by jwntrs** [vmware-tanzu/cartographer#121]
As a developer I want my `Workload` to specify a service account that has permissions to do everything I need in a `SupplyChain`.

This could include things like:
- push/pull images
- push/pull source code
- submit objects to api-server

Related #51

**Comment by DanieldeR** [comment]
The scope of this issue is just limited to:
* Documentation (when a user stamps out an object that can use a service account - we need to document that they should use the workload service account)
* Examples 
* Tests

This is blocked on #51

**Comment by jwntrs** [comment]
closing in favour of #51, will move notes on docs, examples, tests to #51 instead
